css/style.css

```diff
@@ -689,6 +699,42 @@ ul.chars{
     margin: 0 auto;
 }
 
+@media only screen and (max-device-width:1742px) {
+    .page-wide {
+        width: 100%;
+        margin: 0 auto;
+    }
+    a.nav {
+        text-align: center;
+        font-size:0.5em
+    }
+    .nav-pages {
+        float: none;
+    }
+    .brand-logo {
+        display: none;
+    }
+    .brand-link {
+        float: none;
+    }
+    .home-text {
+        display: block !important;
+    }
+    .home-main {
+        width: 100%;
+    }
+    .sidebar {
+        width: 100% !important;
```

Comment:
since this is the css for the entire website you shouldn’t ever really need to use important unless you’re inverting the priority of a selector, but you aren’t doing that